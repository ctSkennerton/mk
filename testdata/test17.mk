
prereq.(\w+):R:
	echo $stem1 $(echo ${stem1})

proto.txt: prereq.txt
	echo $target