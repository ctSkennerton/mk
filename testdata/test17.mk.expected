prereq.txt: echo txt $(echo txt)
proto.txt: echo proto.txt
